packages/eas-cli/src/credentials/manager/ManageAndroidBeta.ts

```diff
@@ -52,7 +59,38 @@ export class ManageAndroid implements Action {
           } else {
             displayAndroidAppCredentials({ appLookupParams, legacyAppCredentials, appCredentials });
           }
-          throw new Error('Not Implemented Yet');
+        }
+        const actions: { value: ActionType; title: string }[] = [
+          {
+            value: ActionType.CreateKeystore,
+            title: 'Set up a new keystore',
+          },
+        ];
+        const { action: chosenAction } = await promptAsync({
+          type: 'select',
+          name: 'action',
+          message: 'What do you want to do?',
+          choices: actions,
+        });
+        if (chosenAction === ActionType.CreateKeystore) {
+          const appLookupParams = getAppLookupParamsFromContext(ctx);
+          const buildCredentialsOrMetadataInput = await new SelectAndroidBuildCredentials(
+            appLookupParams
+          ).runAsync(ctx);
+          const keystore = await new CreateKeystore(appLookupParams.account).runAsync(ctx);
+          if (!('id' in buildCredentialsOrMetadataInput)) {
```

Comment:
Maybe you could make `SelectAndroidBuildCredentials` return a compound object like:
```ts
{
  // this could be an enum
  resultType: 'create_request' | 'existing_credentials';
  // fragment or metadata
  result: AndroidAppBuildCredentialsFragment | AndroidAppBuildCredentialsMetadataInput;
}
```

or you can even do something like this to avoid `as ...` on the call site.:
```
{ resultType: 'create_request; result: AndroidAppBuildCredentialsMetadataInput } | { resultType: 'existing_credentials'; result: AndroidAppBuildCredentialsFragment; }
```


This way you don't need to know the structure of the result - I mean to realize it's metadata when `id` is not present in the object. It also seems to be a lot cleaner solution.

Reply:
this is a very good suggestion, thank you!